Formplayer/utilities/email_conf.py

```diff
@@ -0,0 +1,15 @@
+"""
```

Comment:
we can remove this file as we are no longer using this plugin

Reply:
Done.